### PR TITLE
Add Quarkus Playwright

### PR DESCRIPTION
**Pull request by melloware**
This is a draft until this PR is merged: https://github.com/quarkiverse/quarkus-playwright/pull/95

That switches it from a Test resource to a Runtime resource

### CODE REVIEW

**Comment by gastaldi** [comment]
Funny that this extension never made it to the extension catalog. Maybe exclude the previous versions?

**Comment by melloware** [comment]
Its because its a "test" only depdenency so it has no runtime and this no `quarkus-extension.yaml` which is why we never did it but now it will be possible!

**Comment by melloware** [comment]
OK 2.0.0 is in Maven Central now!

File: extensions/quarkiverse-playwright.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+group-id: "io.quarkiverse.playwright"
+artifact-id: "quarkus-playwright"
+versions:
+- "2.0.0"
+exclude-versions: []
```
Comment (gastaldi) [inline extensions/quarkiverse-playwright.yaml:6]: Add the previous versions here, because they don't have a YAML file

Comment (melloware) [reply extensions/quarkiverse-playwright.yaml:8]: Done!

Comment (gastaldi) [reply extensions/quarkiverse-playwright.yaml:8]: Sorry, I meant exclude the versions, 2.0.0 is the only valid version AFAIK

**Comment by melloware** [comment]
OK see if I did that correctly?

**Review by gastaldi — APPROVED** [review]

**Pull request merged**